test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -681,56 +680,50 @@ function source(::en_ρatke, ::BuoySource, atmos, args)
 end
 
 function source(::en_ρatke, ::DissSource, atmos, args)
-    @unpack env, l_mix, Diss₀ = args.precomputed.turbconv
+    @unpack env, Diss₀ = args.precomputed.turbconv
     gm = args.state
     en = args.state.turbconv.environment
     ρa₀ = gm.ρ * env.a
-    tke_en = enforce_positivity(en.ρatke) / gm.ρ / env.a
+    tke_en = enforce_positivity(en.ρatke) / ρa₀
     return -ρa₀ * Diss₀ * tke_en  # tke Dissipation
 end
 
 function source(::en_ρaθ_liq_cv, ::DissSource, atmos, args)
-    @unpack env, K_h, Diss₀ = args.precomputed.turbconv
-    gm = args.state
+    @unpack Diss₀ = args.precomputed.turbconv
     en = args.state.turbconv.environment
-    ρa₀ = gm.ρ * env.a
-    return -ρa₀ * Diss₀ * en.ρaθ_liq_cv
+    return -Diss₀ * en.ρaθ_liq_cv
```

Comment:
should we use enforce_positivity here as well? 
'' θ_liq_cv  =  enforce_positivity(en.ρaθ_liq_cv) / gm.ρ / env.a ''

Reply:
I don't think we should do it here. This is the dissipation function. If `θ_liq_cv` hasn't been clipped before, having a _positive_ dissipation actually helps bring it to zero.